.gitignore

```diff
@@ -60,3 +60,31 @@ crashlytics-build.properties
 
 # Ignore user settings
 /UserSettings
+
+# Ignore Ez Define Symbols extension
+/[Aa]ssets/Ez/*
+/[Aa]ssets/Plugins/*
+/[Aa]ssets/Ez.meta
+/[Aa]ssets/Plugins.meta
+
+# Un-ignore custom Ez Define Symbols files
+![Aa]ssets/Plugins/Quick
+[Aa]ssets/Plugins/Quick/*
+![Aa]ssets/Plugins/Quick/Editor
+[Aa]ssets/Plugins/Quick/Editor/*
+![Aa]ssets/Plugins/Quick/Editor/Utils
+[Aa]ssets/Plugins/Quick/Editor/Utils/*
+!/[Aa]ssets/Plugins/Quick/Editor/Utils/QUtils.cs
```

Comment:
Nope. The problem you need git to not ignore the parent directory of a file in order to not ignore the file itself.

So I have to carefully tell git to not ignore these directories, but to ignore almost all the individual files in them using this nested directory approach.

I originally attempted the implementation suggested and it still ignores QUtils.cs because /Assets/Plugins/Quick doesn't exist in its perspective, and obviously there can't be files in a directory that doesn't exist, _riiiiiiiight_?

Thank you git. Very good.

Reply:
Well, we could also just manually add the files we want to keep and just generically ignore everything else (see our recent scare a while back)

Reply:
We could manually add them, but I don't want to have to be sent a new preset one of our team members makes over google drive or something and then manually stick it in the right folder from my file explorer.

I'd rather those at least be stored in the repository for the practical purpose of being able to share them among the team with minimal effort.

QUtils.cs is arguably different because it's a one-time fix, unlike presets which will be developed as the project goes on. However, we already have a means of sharing files with each other (the repository) and it's only a 14 KB file (unlike all of our visual assets that we all store in the repository), and the only requirement is that .gitignore has a couple sections that have a bunch of lines that, while looking kinda uggo, function well enough to resolve the issue of distributing QUtils.cs. 

In a manner that's more convenient to our developers than uploading it to google drive or something and giving a link to it so they can download the file and manually place in the right directory themselves, in fact.

Like, there's even comments in .gitignore explaining what the sections do, so it's a non-issue to me.

Reply:
Git tracking is shared across the repo no? So for example if you have a new preset and you manually track it on your end and push it to the branch, then when I pull the branch it'll be added to my index as well

Reply:
...the presets are placed in a folder several layers down in the Plugins folder that I **also** had to make a similar special exception for in .gitignore. It's lines 79-90 of .gitignore in fact.

So... no. The only way to add it on my end and push it to the branch is with the exact same technique I used for QUtils.cs.

Reply:
dab on'em

Reply:
For future reference again, I've altered EZT.cs to use Assets/Resources/Presets/ to store scripting symbol presets now.

.gitignore is clean as a whistle again.